frontend/src/components/createProjectForm.tsx

```diff
@@ -0,0 +1,282 @@
+import useTranslation from "next-translate/useTranslation";
+import { Badge, Col, Row } from "react-bootstrap";
+import styles from "../styles/createProjectForm.module.css";
+import { Field, Form, Formik } from "formik";
+import useSWR from "swr";
+import apiPaths from "../properties/apiPaths";
+import { getAllUsersFromPage } from "../api/calls/userCalls";
+import { capitalize } from "../utility/stringUtil";
+import { ProjectCreationProps, ProjectCreationValues } from "../handlers/createProjectSubmitHandler";
+import { IUser } from "../api/entities/UserEntity";
+import { getAllSkillTypesFromPage } from "../api/calls/skillTypeCalls";
+import { ISkillType } from "../api/entities/SkillTypeEntity";
+import { ChangeEvent, useState } from "react";
+import Image from "next/image";
+
+export const CreateProjectForm = (props: ProjectCreationProps) => {
+    let userResponse = useSWR(apiPaths.users, getAllUsersFromPage);
+
+    let users: IUser[] = userResponse.data || [];
+    let userError: Error = userResponse.error;
+
+    let skillTypeResponse = useSWR(apiPaths.skillTypes, getAllSkillTypesFromPage);
+
+    let skillTypes: ISkillType[] = skillTypeResponse.data || [];
+    let skillTypeError: Error = skillTypeResponse.error;
+
+    const { t } = useTranslation("common");
+
+    const [coaches, setCoaches] = useState<string[] | []>([]);
+    const [hasPickedCoach, setHasPickedCoach] = useState<boolean>(false);
+    const [selectedCoach, setSelectedCoach] = useState<string>("");
+    const [skills, setSkills] = useState<string[] | []>([]);
+    const [selectedSkill, setSelectedSkill] = useState<string>("");
+    const [skillInfos, setSkillInfos] = useState<string[] | []>([]);
+    const [skillInfo, setSkillInfo] = useState<string>("");
+
+    if (userError) {
+        console.log(userError);
+        return null;
+    }
+
+    if (skillTypeError) {
+        console.log(skillTypeError);
+        return null;
+    }
+
+    let skillColorMap = new Map<string, string>();
+
+    for (let skillType of skillTypes) {
+        skillColorMap.set(skillType.name, skillType.colour);
+    }
+
+    function handleChangeSkill(e: ChangeEvent<HTMLInputElement>) {
+        setSelectedSkill(e.target.value);
+    }
+
+    function handleChangeSkillInfo(e: ChangeEvent<HTMLInputElement>) {
+        setSkillInfo(e.target.value);
+    }
+
+    function handleDeleteSkill(index: number) {
+        delete skills[index];
+        delete skillInfos[index];
+        setSkills(skills.filter((value) => value !== undefined));
+        setSkillInfos(skillInfos.filter((value) => value !== undefined));
+    }
+
+    function handleAddSkill() {
+        const newSkill = selectedSkill === "" ? skillTypes[0].name : selectedSkill;
```

Comment:
```suggestion
        const newSkill = selectedSkill ? skillTypes[0].name : selectedSkill;
```
(Multiple occasions)